Makefile

```diff
@@ -8,9 +8,9 @@ flake8:
 	flake8
 
 test:
-	if [[ "$$PYTHON_VERSION" =~ ^(3.6|3.7|3.8|nightly|pypy3)$$ ]] ; then \
+	if [[ "$$PYTHON_VERSION" =~ ^(3.7|3.8|nightly)$$ ]] ; then \
 		py.test -v $(PYTEST_ARGS) $(PYTEST_MARKER) $(PYTEST_JUNIT); \
-	elif [[ "$$PYTHON_VERSION" =~ ^3\.5$$ ]] ; then \
+	elif [[ "$$PYTHON_VERSION" =~ ^(3.5|3.6|pypy3)$$ ]] ; then \
```

Comment:
it might make sense to add a comment here regarding pypy3 that it should be removed once it is on par with Python 3.7 (currently it's 3.6, not sure what their timeline is for 3.7+)